themes/src/main/resources-community/theme/base/account/messages/messages_es.properties

```diff
@@ -1,3 +1,5 @@
+# encoding: utf-8
+
```

Comment:
This should already be the default, see #21755. You should be able to safely remove this.
```suggestion
```